telepresence/outbound/vpn.py

```diff
@@ -136,10 +143,12 @@ def k8s_resolve(
                 )
             )
     else:
-        resolved_ips = []
+        hostname_ips = []
 
-    for host, ip in zip(hostnames, resolved_ips):
-        ipcache[host] = ip
+    resolved_ips: List[str] = []
```

Comment:
This syntax does not work for Python 3.5, which Telepresence must support.

Reply:
Use something like this instead:

    resolved_ips = []  # type: List[str]

Reply:
(I'm annoyed that CI did not catch that, but I will fix that elsewhere...)

Reply:
Aight changed :D just had the linter complain on me as it wanted a type hint